pkg/flags/flags.go

```diff
@@ -66,16 +65,25 @@ func setLogLevel(logLevel string) {
 	}
 	log.SetLevel(level)
 
-	if level >= log.DebugLevel {
-		flag.Set("stderrthreshold", "INFO")
-		flag.Set("logtostderr", "true")
-		flag.Set("v", "6")
-		// pipe klog entries to logrus
-		klog.SetOutput(log.StandardLogger().Writer())
-	}
-
-	if level >= log.TraceLevel {
-		flag.Set("v", "12") // At 7 and higher, authorization tokens get logged.
+	// Based on k8s logging conventions, except for 'tracing' that we bump to 7
```

Comment:
That doc only talks about log levels up to 5 but we previously had trace mapped to 12.  Is there any difference between level 7 and 12?  If so, we probably want to have a mode which logs at the maximum possible level.

We also previously used level 6 for debug because it was the maximum possible level that avoided logging auth tokens.  It seems like that's a useful level.  Would mapping DEBUG to 6 instead of 4 be too verbose?

Reply:
Our max value for `v` used to be 6, and it got bumped to 12 in d540e16c8bfd4898d7851e6c1899bd3e43e220f5. But grepping client-go's source, I can't find any usage above 10. So what do you think about setting `trace` at 10 and `debug` at 6?

Reply:
sounds good to me!